package.json

```diff
@@ -36,6 +36,7 @@
     "cnbuilder": "^3.1.0",
     "framer-motion": "^10.15.0",
     "next": "13.4.9",
+    "react-player": "^2.13.0",
```

Comment:
Same popular react-player package as the one we used for Giving.
https://github.com/CookPete/react-player